c2corg_ui/__init__.py

```diff
@@ -68,3 +73,19 @@ def _add_static_view(config, name, path):
     )
     config.add_cache_buster(path, version_cache_buster)
     CACHE_PATH.append(name)
+
+
+class NotFound():
+
+    notFound = 'c2corg_ui:templates/404.html'
+
+    def __init__(self, request):
+        self.settings = request.registry.settings
+
+    @view_config(context=HTTPInternalServerError, renderer=notFound)
+    @view_config(context=HTTPBadRequest, renderer=notFound)
+    @notfound_view_config(renderer=notFound)
+    def notfound(self):
+        return {
```

Comment:
Yep that sounds relevant.
Just not sure how one can get the response status code and forward it in this ``notfound`` view,